web/massastation/src/components/SidePanels/Upload/Upload.tsx

```diff
@@ -164,6 +175,39 @@ export default function Upload() {
     });
   }
 
+  const { data: accounts = [] } = useResource<AccountObject[]>(
+    `${URL.WALLET_BASE_API}/${URL.WALLET_ACCOUNTS}`,
+  );
+
+  const accountsItems = accounts.map((account) => ({
+    icon: <Identicon username={account.nickname} size={32} />,
+    item: account.nickname,
+    onClick: () => setNickname(account.nickname),
+  }));
+
+  const selectedAccountKey: number = parseInt(
+    Object.keys(accounts).find(
+      (_, idx) => accounts[idx].nickname === nickname,
+    ) || '0',
+  );
+
+  const existingAccount: boolean = accounts.length > 0;
+
+  const [pluginWalletIsInstalled, setPluginWalletIsInstalled] = useState(false);
+
+  const { data: plugins, isSuccess } =
+    useResource<PluginHomePage[]>('plugin-manager');
+
+  useEffect(() => {
+    if (isSuccess) {
+      plugins.forEach((plugin) => {
+        if (plugin.name === 'Massa Wallet') {
```

Comment:
I guess @Thykof created a const with that name here. Look for MASSA_WALLET in the codebase.